README.md

```diff
@@ -37,17 +37,58 @@ For Python projects, you can use the tools in this script with the following
 commands:
 
 ```yaml
-stages:
-  - test
-  - name: deploy
-    if: (branch = master OR tag IS present) AND type != pull_request
+version: ~> 1.0
+
+env:
+  global:
+    # doctr generated secure variable for documentation upload
+    - secure: "<your secure here>"
+    # enable the usage of versions menu which allow versioning of the docs
+    # pages and not only the master branch
+    - DOCTR_VERSIONS_MENU="1"
+    # Dependency files used to build the documentation (space separated)
+    - DOCS_REQUIREMENTS="dev-requirements.txt requirements.txt"
+    # Options to be passed to flake8 for package linting. Usually this is just
+    # the package name but you can enable other flake8 options via this config
+    - PYTHON_LINT_OPTIONS="my_package"
+
+    # The name of the conda package
+    - CONDA_PACKAGE="my_package"
+    # The folder containing the conda recipe (meta.yaml)
+    - CONDA_RECIPE_FOLDER="conda-recipe"
+    # Extra dependencies needed to run the tests which are not included
+    # at the recipe and dev-requirements.txt. E.g. PyQt
+    - CONDA_EXTRAS="pip pyqt=5 happi"
+    # Requirements file with contents for tests dependencies
+    - CONDA_REQUIREMENTS="dev-requirements.txt"
+
+    # Extra dependencies needed to run the test with Pip (similar to
+    # CONDA_EXTRAS) but for pip
+    - PIP_EXTRAS="PyQt5 happi"
+
+jobs:
+  allow_failures:
+    # This makes the PIP based Python 3.6 optional for passing.
+    # Remove this block if passing tests with PIP is mandatory for your
+    # package
+    - name: "Python 3.6 - PIP"
 
 import:
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-linter.yml
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester.yml
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/pypi-upload.yml
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/doctr-upload.yml
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/anaconda-upload.yml
+  # If your project requires X11 leave the following import
+  - pcdshub/pcds-ci-helpers:travis/shared_configs/setup-env-ui.yml
+  # This import enables a set of standard python jobs including:
+  # - Build
+  #   - Anaconda Package Build
+  # - Tests
+  #   - Linter
+  #   - Documentation
+  #   - Python 3.6 - PIP based
+  #   - Python 3.6, 3.7 & 3.8 - Conda base
+  # - Deploy
+  #   - Documentation using doctr
+  #   - Conda Package - uploaded to pcds-dev and pcds-tag
+  #   - PyPI
+  - pcdshub/pcds-ci-helpers:travis/shared_configs/standard-python-conda.yml
```

Comment:
I like this being the default, recommended option. 

Could we also have a section that lists the other (currently) available options like you had before?

Then for users they can simply delete the unneeded commented out lines (or change to using them).

Reply:
Great idea! I will add them back in.

Reply:
Addressed via https://github.com/pcdshub/pcds-ci-helpers/pull/34/commits/9c24f3c803c480d26d9eed14f7e753b690bc29c0